meta-lmp-base/recipes-samples/images/lmp-feature-xeno4.inc

```diff
@@ -0,0 +1,4 @@
+# Xenomai4 packages
+CORE_IMAGE_BASE_INSTALL += " \
+    xenomai4 \
+"
```

Comment:
We can also change the preferred kernel here 
```
PREFERRED_PROVIDER_virtual/kernel:mx8mm-nxp-bsp = "linux-lmp-fslc-imx-xeno4"
```

Reply:
With that the user only need to add `xeno4` to the `MACHINE_FEATURES` list to enable the Xenomai4
```
echo 'MACHINE_FEATURES:append = " xeno4"' >> conf/local.conf 
```